### PR TITLE
Add Job Comments

### PR DESCRIPTION
**Pull request by orwigeg**

### CODE REVIEW

File: doc/compiled.json [hunk 1]
```diff
@@ -21686,6 +21690,416 @@
...
+          }
+        },
+        "x-code-samples": [
+          {
+            "lang": "Curl",
+            "source": "curl \"https://api.phrase.com/v2/projects/:project_id/jobs/:job_id/comments/:id\" \\\n  -u USERNAME_OR_ACCESS_TOKEN \\\n  -X PATCH \\\n  -d '{\"message\":\"Some message...\"}' \\\n  -H 'Content-Type: application/json'"
+          },
+          {
+            "lang": "CLI v2",
+            "source": "phrase comments update \\\n--project_id <project_id> \\\n--job_id <job_id> \\\n--id <id> \\\n--data '{\"message\": \"Some message...\"}' \\\n--access_token <token>"
```
Comment (theSoenke) [inline doc/compiled.json:21828]: Not sure whether this is wrapped in `data` or you could just use `--message "comment"` directly

File: doc/compiled.json [hunk 2]
```diff
@@ -21686,6 +21690,416 @@
...
+          }
+        },
+        "x-code-samples": [
+          {
+            "lang": "Curl",
+            "source": "curl \"https://api.phrase.com/v2/projects/:project_id/jobs/:job_id/comments\" \\\n  -u USERNAME_OR_ACCESS_TOKEN \\\n  -X POST \\\n  -d '{\"message\":\"Some message...\"}' \\\n  -H 'Content-Type: application/json'"
+          },
+          {
+            "lang": "CLI v2",
+            "source": "phrase comments create \\\n--project_id <project_id> \\\n--job_id <job_id> \\\n--data '{\"message\": \"Some message...\"}' \\\n--access_token <token>"
```
Comment (theSoenke) [inline doc/compiled.json:22081]: Same here with the message. I'd suggest to build the CLI and check how the parameters look like just to be sure

File: doc/compiled.json (hunk 1 above)
Comment (orwigeg) [reply doc/compiled.json:21828]: Oh I think I understand
This is how is looks for `paths/comments/update.yaml` 
```- lang: CLI v2
  source: |-
    phrase comments update \
    --project_id <project_id> \
    --key_id <key_id> \
    --id <id> \
    --data '{"branch":"my-feature-branch", "message": "Some message..."}' \
    --access_token <token>
requestBody:
  required: true
  content:
    application/json:
      schema:
        type: object
        title: comment/update/parameters
        properties:
          branch:
            description: specify the branch to use
            type: string
            example: my-feature-branch
          message:
            description: Comment message
            type: string
            example: Some message...
```
And this is how it looks for `paths/job_comments/update.yaml
```
- lang: CLI v2
  source: |-
    phrase comments update \
    --project_id <project_id> \
    --job_id <job_id> \
    --id <id> \
    --data '{"message": "Some message..."}' \
    --access_token <token>
requestBody:
  required: true
  content:
    application/json:
      schema:
        type: object
        title: job_comment/update/parameters
        properties:
          message:
            description: Comment message
            type: string
            example: Some message...
```
so it should be the same

Comment (theSoenke) [reply doc/compiled.json:21828]: Just checked and looks all good :+1:

File: doc/compiled.json [hunk 3]
```diff
@@ -21686,6 +21690,416 @@
           }
         }
       }
+    },
+    "/projects/{project_id}/jobs/{job_id}/comments/{job_comment_id}": {
```
[the remark this answers is not in the document]
Comment (theSoenke) [reply doc/compiled.json:21694]: I think you would need to regenerate this compiled file to update the path here

**Comment by sbruhns** [comment]
Looks good. Please make sure that libs and clients are still building and working.

**Review by sbruhns — APPROVED** [review]

**Pull request merged**